src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -2540,45 +2523,47 @@ namespace Akka.Dispatch.MessageQueues
         public UnboundedMessageQueue() { }
         public int Count { get; }
         public bool HasMessages { get; }
-        public void Enqueue(Akka.Actor.Envelope envelope) { }
+        public void CleanUp(Akka.Actor.IActorRef owner, Akka.Dispatch.MessageQueues.IMessageQueue deadletters) { }
+        public void Enqueue(Akka.Actor.IActorRef receiver, Akka.Actor.Envelope envelope) { }
         public bool TryDequeue(out Akka.Actor.Envelope envelope) { }
     }
     public class UnboundedPriorityMessageQueue : Akka.Dispatch.MessageQueues.BlockingMessageQueue
     {
-        public UnboundedPriorityMessageQueue(System.Func<object, int> priorityGenerator) { }
+        public UnboundedPriorityMessageQueue(int initialCapacity) { }
+        public UnboundedPriorityMessageQueue(System.Func<object, int> priorityGenerator, int initialCapacity) { }
         protected override int LockedCount { get; }
         protected override void LockedEnqueue(Akka.Actor.Envelope envelope) { }
         protected override bool LockedTryDequeue(out Akka.Actor.Envelope envelope) { }
     }
 }
 namespace Akka.Dispatch.SysMsg
 {
-    public sealed class ActorTask : Akka.Actor.INoSerializationVerificationNeeded, Akka.Dispatch.SysMsg.ISystemMessage
+    public sealed class ActorTask : Akka.Dispatch.SysMsg.SystemMessage
```

Comment:
`ISystemMessage` still exists, but now all sysmsg instances subclass from `SystemMessage` - an implementation class which includes all of the capabilities we need for linking system messages into lists.